src/libraries/Microsoft.PowerFx.Core/Public/Config/ReadOnlySymbolTable.cs

```diff
@@ -93,6 +93,20 @@ internal static ReadOnlySymbolTable NewDefault(IEnumerable<TexlFunction> coreFun
             return s;
         }
 
+        // Helper to create a ReadOnly symbol table around a set of core functions.
+        // Important that this is mutable so that it can be changed across engines. 
+        public SymbolTable GetMutableCopy()
+        {
+            var s = new SymbolTable();
```

Comment:
>SymbolTable [](http://example.com/codeflow?start=24&length=11)

nit: we should copy DebugName too. 
If we just copy functions, maybe 

s.DebugName = this.DebugName + " (Functions only)";

Reply:
Also - what about Parent. 
We could:
s._parent   = this.Parent 

I don't think we should also copy the parent. But a caller could do the deep copy themselves if they really wanted that.

Reply:
Parent is not being copied.